src/dune

```diff
@@ -2,8 +2,9 @@
  (name patch)
  (synopsis "Patch purely in OCaml")
  (public_name patch)
- (modules patch)
- (wrapped false))
+ (modules patch patch_lib fname))
```

Comment:
My suggestion would be dropping this and just moving the executable into a different folder, thus new modules would be picked up automatically.

Reply:
this can be done later